server/utils/assessmentUtils.ts

```diff
@@ -13,6 +13,7 @@ import { embeddedSummaryListItem } from './checkYourAnswersUtils'
 import reviewSections from './reviewUtils'
 import Apply from '../form-pages/apply'
 import { kebabCase } from './utils'
+import { suitabilityAssessmentSections as suitabilityAssessmentQuestions } from '../form-pages/assess/assessApplication/suitablityAssessment/suitabilityAssessment'
```

Comment:
Could we not have this as a property on the form?

Reply:
Yeah that makes sense